cloudflare.php

```diff
@@ -13,7 +13,6 @@
     exit;
 }
 
-define('CLOUDFLARE_MIN_WP_VERSION', '3.4');
```

Comment:
I moved it to Hooks cause thats where the version check happened.  I don't care where the constant lives though.  Looks like you moved it back in the other PR, thats fine.